### ISSUE TITLE
Show path in inspector

### ISSUE THREAD

**Issue by treydavis** [NicolasPetton/Indium#212]
I'd like to be able to see the path I've taken while exploring a deeply-nested object with `indium-inspector`:

`path.taken[3].through.a.deeply[0].nested.object`

**Comment by NicolasPetton** [comment]
Trey Davis <notifications@github.com> writes:

> I'd like to be able to see the path I've taken while exploring a deeply-nested object with `indium-inspector`:
>
> `path.taken[3].through.a.deeply[0].nested.object`

Excellent idea!

**Comment by NicolasPetton** [comment]
@treydavis Just so we're clear, you do not want a history of the navigation in the inspector, right?

**Comment by treydavis** [comment]
Right, I was imagining a feature similar to what I've seen in `cider-inspect-mode`:
<img width="429" alt="Screen Shot 2019-09-05 at 11 11 51 AM" src="https://user-images.githubusercontent.com/1691239/64367945-0d9b3980-cfce-11e9-9ae6-7a77958869a3.png">

In Cider, I found that the feature is implemented on the server-side in https://github.com/clojure-emacs/cider-nrepl. It keeps an "inspector" map with a `:path` key: 
``` clj
{:value nil, :stack [], :path [], :pages-stack [],
          :current-page 0, :rendered '()}
```
https://github.com/clojure-emacs/orchard/blob/master/src/orchard/inspect.clj#L65-L70
